index.html

```diff
@@ -0,0 +1,747 @@
+            <div>
+                <ul class="middle-1">
+                    <li><strong>THE UNIVERSE</strong> is just under 14 billion years old. From our position in the Milky Way galaxy, we can observe a sphere that is now about 92 billion light-years across. But there's a mystery. Wherever we look, the universe has an even temperature.</li>
+                    <li>
+                        <strong>ENOUGH TIME</strong>  The universe is not old enough for light to have traveled the vast distance from one side of the universe to the other, and there has not been enough time for scattered patches of hot and cold to mix into an even temperature.
+                    </li>
+                    <li>
+                        <strong>COFFEE</strong> At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled. 
+                    </li>
+                    <li>
+                        <strong>COFFEE EVERYWHERE</strong>  Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to look for another explanation.
+                    </li>
+                    </ul>
+                    <ul class="middle-2">
+                    <li>
+                        <strong>STILL NOT ENOUGH TIME</strong> There has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might explain their even temperature.
+                    </li>
+                    <li>
+                        <strong>INFLATION</strong>  solves this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.
+                    </li>
+                    <li>
+                        <strong>FLUCTUATION</strong>  Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.
+                    </li>
+                    <li>
+                        <strong>EXPANSION</strong>  Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.
+                    </li>
+                </ul>
+        </div>
+        </section>
+        <article style="margin-top: 2em;">
```

Comment:
Please, remove inline styling, it is considered a bad practice, move it to your CSS files.